vendor/autoload.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
remove vendor folder and put in .gitignore

Reply:
done